pre-processing/primod/typing.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+
+
+Int = np.int_
+Float = np.float_
```

Comment:
```suggestion
Float = np.float64
```

Will be deprecated.
One more reason IMO to adapt the whole of primod